R/map.R

```diff
@@ -179,21 +204,60 @@ map_ <- function(.type,
   )
 }
 
+mmap_ <- function(.x, .f, .args, .parallel, .progress, .type, error_call) {
```

Comment:
Maybe this should live in the parallelisation file?

Reply:
Yes, I guess it's used by several of the files. Moved in [d348652](https://github.com/tidyverse/purrr/pull/1163/commits/d348652b4e54a6897a50e486d98f46f945d514df).

Reply:
Actually due to https://github.com/tidyverse/purrr/pull/1163#discussion_r1927130312 I've moved it back so that the parallel function versions are just directly below the normal ones. It's easier so that everything is in one place.